service/app/modules/OnRequestHandler.java

```diff
@@ -115,7 +115,7 @@ public Action createAction(Http.Request request, Method method) {
     return CompletableFuture.completedFuture(Results.status(responseCode, Json.toJson(resp)));
   }
 
-  private void intializeRequestInfo(Http.Request request, String userId) {
+  public void intializeRequestInfo(Http.Request request, String userId) {
```

Comment:
What is the purpose behind modifier change?

Reply:
done